hs/app/reachpc/ReachPC/Config.hs

```diff
@@ -63,12 +72,92 @@ data RchToml = RchToml
   , rtml_organization :: !(Maybe String)
   } deriving (Show)
 
+emptyRchToml :: RchToml
+emptyRchToml = RchToml Nothing Nothing Nothing Nothing Nothing
+
+-- helper for interactiveConfigCreate
+data PromptAnswer = Unset | Unchanged | Str String
+
+-- Code for the `reach config` subcommand
+interactiveConfigCreate :: IO ()
```

Comment:
I think it is better to have this code in Main. If we are to have two files (blech), then I think it makes sense for there to be one "data" file and the other "UI" file